src/__tests__/render.test.js

```diff
@@ -382,3 +382,25 @@ test('renders options.wrapper around updated node', () => {
     </RCTSafeAreaView>
   `);
 });
+
+test('returns container', () => {
+  const { container } = render(<View testID="inner" />);
+
+  expect(container).toBeDefined();
+  expect(container.type).toBe(View);
+  expect(container.props.testID).toBe('inner');
+});
+
+test('returns wrapped component as container', () => {
+  const WrapperComponent = ({ children }) => (
+    <SafeAreaView testID="wrapper">{children}</SafeAreaView>
+  );
+
+  const { container } = render(<View testID="inner" />, {
+    wrapper: WrapperComponent,
+  });
+
+  expect(container).toBeDefined();
+  expect(container.type).toBe(WrapperComponent);
+  expect(container.props.testID).not.toBeDefined();
```

Comment:
I though that it should be `"wrapper"` here.

Reply:
We're not currently normalizing container to return a host component, so the actual instance of `WrapperComponent` is returned, hence no testID